blaze-core/src/main/scala-2.13/org/http4s/blazecore/util/ParasiticExecutionContextCompat.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.blazecore.util
+
+import org.http4s.blaze.util.Execution
+
+import scala.concurrent.ExecutionContext
+
+private[util] trait ParasiticExecutionContextCompat {
+  final def parasitic: ExecutionContext = Execution.trampoline
```

Comment:
Huh, shouldn't this be `ExecutionContext.parasitic` on Scala 2.13?

Reply:
Hmm, seems like I saw the `Execution.parasitic` indeed for 2.13 in Ross's PR.

Reply:
Uh-oh, this was for the Scala 3 https://github.com/http4s/http4s/pull/6118/files#diff-132f6ad9670a22754e1983195601c792e110429f8b1e70f1b3f992fbed456409R22. Probably, it worth fixing for downstream too.

Reply:
@danicheg yes. While you fix that, the async deprecation versions need to be fixed too 😅

Reply:
Let's wait a bit for @rossabaker opinion on the topic.

Reply:
Fuck.  Well, they behave about the same, but let's get it fixed as part of the next train.

Reply:
Done #6145